dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/ServiceImpl.java

```diff
@@ -371,10 +429,16 @@ public <T> T getPort(EndpointReferenceType endpointReference,
 
     public <T> T getPort(EndpointReferenceType endpointReference, Class<T> type,
                          WebServiceFeature... features) {
+        
+        boolean isFinestEnabled = LOG.isLoggable(Level.FINEST);   // Liberty Change issue #26529
         endpointReference = EndpointReferenceUtils.resolve(endpointReference, bus);
         QName serviceQName = EndpointReferenceUtils.getServiceName(endpointReference, bus);
         String portName = EndpointReferenceUtils.getPortName(endpointReference);
-
+        if (isFinestEnabled) {
+            LOG.finest("Service QName(serviceQName): " + serviceQName);  // Liberty Change issue #26529
```

Comment:
I would change this to "LOG.finest("Service QName: " + serviceQName);" and "LOG.finest("Port Name: " + portName); "